src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-details/rgw-storage-class-details.component.html

```diff
@@ -0,0 +1,96 @@
+<ng-container *ngIf="!!selection">
+  <cds-tabs type="contained"
+            theme="light">
+    <cds-tab heading="Details">
+      <table
+        class="cds--data-table--sort cds--data-table--no-border cds--data-table cds--data-table--md"
+        data-testid="rgw-storage-details"
+      >
+        <tbody>
+          <tr>
+            <td i18n
+                class="bold">
+              Target Path
+              <cd-helper class="text-pre-wrap">
+                <span>
+                  The target path specifies a prefix to which the source bucket-name/object-name is
+                  appended.
+                </span>
+              </cd-helper>
+            </td>
+            <td>{{ selection?.target_path }}</td>
+          </tr>
+          <tr>
+            <td i18n
+                class="bold">
+              Access key
+              <cd-helper class="text-pre-wrap">
+                <span>
+                  Access key is the remote cloud S3 access key used for a specific connection.
+                </span>
+              </cd-helper>
+            </td>
+            <td>
+              <div cdsCol
+                   [columnNumbers]="{ md: 4 }"
+                   class="d-flex">
+                <input
+                  cdsPassword
+                  type="password"
+                  readonly
+                  id="access_key"
+                  [value]="selection?.access_key"
+                />
+                <button type="button"
+                        class="btn btn-light"
+                        cdPasswordButton="access_key"></button>
+                <cd-copy-2-clipboard-button source="access_key"> </cd-copy-2-clipboard-button>
+              </div>
+            </td>
+          </tr>
+          <tr>
+            <td i18n
+                class="bold">
+              Secret key
+              <cd-helper class="text-pre-wrap">
+                <span> Secret is the secret key for the remote cloud S3 service. </span>
```

Comment:
i18n

Reply:
You should be adding i18n on the first parent (closest to text). So here we can add to span